Broken update to 9.20

### Describe the Bug

Here is the error reported in the console when trying to update.

node:internal/modules/cjs/loader:988
  throw err;
  ^

Error: Cannot find module '@authenio/samlify-node-xmllint'
Require stack:
- /home/david/directus/bamboo/node_modules/directus/dist/auth/drivers/saml.js
- /home/david/directus/bamboo/node_modules/directus/dist/auth/drivers/index.js
- /home/david/directus/bamboo/node_modules/directus/dist/auth.js
- /home/david/directus/bamboo/node_modules/directus/dist/services/authentication.js
- /home/david/directus/bamboo/node_modules/directus/dist/services/index.js
- /home/david/directus/bamboo/node_modules/directus/dist/middleware/respond.js
- /home/david/directus/bamboo/node_modules/directus/dist/controllers/activity.js
- /home/david/directus/bamboo/node_modules/directus/dist/app.js
- /home/david/directus/bamboo/node_modules/directus/dist/server.js
- /home/david/directus/bamboo/node_modules/directus/dist/cli/index.js
- /home/david/directus/bamboo/node_modules/directus/dist/cli/run.js
- /home/david/directus/bamboo/node_modules/directus/cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
    at Function.Module._load (node:internal/modules/cjs/loader:833:27)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/david/directus/bamboo/node_modules/directus/dist/auth/drivers/saml.js:30:32)
    at Module._compile (node:internal/modules/cjs/loader:1155:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1209:10)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Module.require (node:internal/modules/cjs/loader:1057:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/david/directus/bamboo/node_modules/directus/dist/auth/drivers/saml.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/auth/drivers/index.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/auth.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/services/authentication.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/services/index.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/middleware/respond.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/controllers/activity.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/app.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/server.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/cli/index.js',
    '/home/david/directus/bamboo/node_modules/directus/dist/cli/run.js',
    '/home/david/directus/bamboo/node_modules/directus/cli.js'
  ]
}


### To Reproduce

After running npm update was successful.
I ran: npx directus database migrate:latest
and the error was thrown

### Errors Shown

_No response_

### What version of Directus are you using?

9.20

### What version of Node.js are you using?

16.14.2

### What database are you using?

postgres 14

### What browser are you using?

chrome

### How are you deploying Directus?

CLI/npx

--- a/api/package.json
+++ b/api/package.json
@@ -76,6 +76,7 @@
 		"README.md"
 	],
 	"dependencies": {
+		"@authenio/samlify-node-xmllint": "2.0.0",
 		"@aws-sdk/client-ses": "3.190.0",
 		"@directus/app": "workspace:*",
 		"@directus/drive": "workspace:*",
@@ -179,7 +180,6 @@
 	},
 	"gitHead": "24621f3934dc77eb23441331040ed13c676ceffd",
 	"devDependencies": {
-		"@authenio/samlify-node-xmllint": "2.0.0",
 		"@otplib/preset-default": "12.0.1",
 		"@types/async": "3.2.15",
 		"@types/busboy": "1.5.0",

